helm/templates/networkpolicy.yaml

```diff
@@ -2,11 +2,11 @@
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
-  name: ingress-network-policy-callisto-timecard
+  name: ingress-network-policy-{{ .Release.Name }}
```

Comment:
Would a prefix be better so that when ordering resources from the same release will appear together. It might not be an issue when we only have a single resource but I think it's part of the recommended best practices
Couldn't find a link under best practices/ guidelines but found where I read this
https://helm.sh/docs/chart_template_guide/getting_started/#adding-a-simple-template-call

Reply:
Yeah I think that's better, updated now